heatexchanger/rectpipe.py

```diff
@@ -111,7 +111,7 @@ def setup(self, Nsegments, Nfins, fluid, increasingT):
                 (Pf/Pf_ref)**0.155 >= (0.475*(Re[-1]/Re_ref)**0.00121
                                        + 0.0338*(Re[-1]/Re_ref)**-0.336),
 
-                D >= fr * Nfins * A_seg[0]
+                D >= fr * Nfins * A[0]
```

Comment:
Oh, and I can't get it to tile with this as `A[0]`, which is maddening, so master will continue to use `A_seg[0]` :(